src/modules/search.js

```diff
@@ -96,6 +97,8 @@ function createSearchUrl(query, parameters, options) {
 class Search {
   constructor(options) {
     this.options = options;
+    this.module = 'search';
+    this.eventDispatcher = new EventDispatcher(options && options.eventDispatcher);
```

Comment:
Maybe we could do 
``` javascript
this.options = options || {};
```
so that we can do
```
this.eventDispatcher = new EventDispatcher(options.eventDispatcher);
```